rpcs3/Emu/system_progress.cpp

```diff
@@ -92,7 +92,7 @@ void progress_dialog_server::operator()()
 				{
 					// Abort everything
 					sys_log.notice("Aborted progress dialog");
-					Emu.Stop();
+					Emu.Kill();
```

Comment:
there is an edge case. namely when there are ppu modules compiled during gameplay and this dialog is open.